src/impl/EspNowNode.cpp

```diff
@@ -159,7 +169,18 @@ bool EspNowNode::setup() {
     request.discovery_challenge = esp_random();
 
     int8_t retries = NUMBER_OF_RETRIES_FOR_DISCOVERY_REQUEST;
+
+    uint8_t channels[] = WIFI_SCAN_CHANNELS;
```

Comment:
Instead of assigning, you could declare `uint8_t WIFI_SCAN_CHANNELS[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14};` at the top instead of the existing `WIFI_SCAN_CHANNELS` define.

Reply:
@jamienz And I suggest doing this as well.